packages/client/src/components/post-form.vue

```diff
@@ -239,16 +239,16 @@ if (props.reply && props.reply.text != null) {
 	for (const x of extractMentions(ast)) {
 		const mention = x.host ?
 											`@${x.username}@${toASCII(x.host)}` :
-											(otherHost == null || otherHost == host) ?
+											(otherHost == null || otherHost === host) ?
 												`@${x.username}` :
 												`@${x.username}@${toASCII(otherHost)}`;
 
 		// 自分は除外
-		if ($i.username == x.username && x.host == null) continue;
-		if ($i.username == x.username && x.host == host) continue;
+		if ($i.username === x.username && x.host == null) continue;
+		if ($i.username === x.username && x.host === host) continue;
 
 		// 重複は除外
-		if (text.indexOf(`${mention} `) != -1) continue;
+		if (text.indexOf(`${mention} `) !== -1) continue;
```

Comment:
I think this can be simplified?
```suggestion
		if (text.includes(`${mention} `)) continue;
```